Campaign_Manager/tests.py

```diff
@@ -1,3 +1,81 @@
 from django.test import TestCase
+from .models import *
+from Character_Builder.models import *
 
-# Create your tests here.
+# Campaign test cases
+class CampaignCreateTestCase(TestCase):
+	def setUp(self):
+		Campaign.objects.create(campaignName="The Mountain")
+
+	def test_campaign_name(self):
+		campaign = Campaign.objects.get(campaignName="The Mountain")
+		self.assertEqual(campaign.campaignName, "The Mountain")
+
+	def test_campaign_id(self):
+		campaign = Campaign.objects.get(campaignName="The Mountain")
+		self.assertEqual(campaign.campaignID, 1)
+
+# CampaignDM test cases
+class CampaignDMCreateTestCase(TestCase):
+	def setUp(self):
+		Campaign.objects.create(campaignName="The Mountain")
+		Character.objects.create(characterName="Malikar", alignment="Lawful Evil", size="Medium")
+		camp = Campaign.objects.get(campaignName="The Mountain")
+		char = Character.objects.get(characterName="Malikar")
+		CampaignDM.objects.create(character=char, campaign=camp)
+
+	def test_campaign_dm_character(self):
+		char = Character.objects.get(characterName="Malikar")
+		dm = CampaignDM.objects.get(character=char)
+		self.assertEqual(dm.character.characterName, "Malikar")
+
+	def test_campaign_dm_id(self):
+		char = Character.objects.get(characterName="Malikar")
+		dm = CampaignDM.objects.get(character=char)
+		self.assertEqual(dm.campaignDMID, 1)
+
+	def test_campaign_dm_campaign(self):
+		char = Character.objects.get(characterName="Malikar")
+		camp = Campaign.objects.get(campaignName="The Mountain")
+		dm = CampaignDM.objects.get(character=char)
+		self.assertEqual(dm.campaign, camp)
+
+class PartyCreateTestCase(TestCase):
+	def setUp(self):
```

Comment:
use the camp variable you create here. Also, move the party variable to setUp to avoid redundancy

Reply:
^^